### ISSUE TITLE
yolov4 cont train

### ISSUE THREAD

**Issue by bsysman** [AlexeyAB/darknet#5590]
then train yolov4,load model success and immediately stop,log:
nms_kind: greedynms (1), beta = 0.600000 
 151 route  147 		                           ->   26 x  26 x 256 
 152 conv    512       3 x 3/ 2     26 x  26 x 256 ->   13 x  13 x 512 0.399 BF
 153 route  152 116 	                           ->   13 x  13 x1024 
 154 conv    512       1 x 1/ 1     13 x  13 x1024 ->   13 x  13 x 512 0.177 BF
 155 conv   1024       3 x 3/ 1     13 x  13 x 512 ->   13 x  13 x1024 1.595 BF
 156 conv    512       1 x 1/ 1     13 x  13 x1024 ->   13 x  13 x 512 0.177 BF
 157 conv   1024       3 x 3/ 1     13 x  13 x 512 ->   13 x  13 x1024 1.595 BF
 158 conv    512       1 x 1/ 1     13 x  13 x1024 ->   13 x  13 x 512 0.177 BF
 159 conv   1024       3 x 3/ 1     13 x  13 x 512 ->   13 x  13 x1024 1.595 BF
 160 conv     24       1 x 1/ 1     13 x  13 x1024 ->   13 x  13 x  24 0.008 BF
 161 yolo
[yolo] params: iou loss: ciou (4), iou_norm: 0.07, cls_norm: 1.00, scale_x_y: 1.05
nms_kind: greedynms (1), beta = 0.600000 
Total BFLOPS 59.578 
avg_outputs = 490041 
 Allocate additional workspace_size = 111.05 MB 
Loading weights from data/model/37.data...
 seen 32, trained: 1025536 K-images (16024 Kilo-batches_64) 

 Warning: Unexpected end of wights-file! l.biases - l.index = 0 
Done! Loaded 0 layers from weights-file 
Learning Rate: 0.001, Momentum: 0.949, Decay: 0.0005
Saving weights to results/37_final.weights
 Create 28 permanent cpu-threads 

my cfg file is :
[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.949
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 10000
policy=steps
steps=8000,9000
scales=.1,.1

**Comment by bsysman** [comment]
@AlexeyAB

**Comment by shengyuqing** [comment]
same problem

**Comment by Chimdee** [comment]
adding `-clear` will work to cut out classification layers from the pretrained weights.

**Comment by AlexeyAB** [comment]
> Loading weights from data/model/37.data...

You are using incorrect training command.
